packages/@aws-cdk/aws-rds/lib/props.ts

```diff
@@ -148,10 +148,24 @@ export interface CredentialsFromUsernameOptions {
  * Username and password combination
  */
 export abstract class Credentials {
+  /**
+   * Creates Credentials for the given username, and optional password and key.
+   * If no password is provided, one will be generated and stored in SecretsManager.
+   */
+  public static fromFixedUsername(username: string, options: CredentialsFromUsernameOptions = {}): Credentials {
```

Comment:
The default is currently handled in the instance/cluster code when `credentials` is not specified. To avoid a breaking change it still falls back to `fromUsername()`.

Reply:
I wonder whether it makes sense for customers to do something like this:

```ts
new rds.DatabaseCluster(this, 'Cluster', {
  // ...
  credentials: rds.Credentials.fromFixedUsername({ excludeCharacters: '!&*^#@()' }),
});
```

"I don't care about the username - just use the default, but I want to change something about the other properties"

Does something like this make sense?

Reply:
Since `Credentials` has a `username` property that cannot be undefined, I think this change means that we need to make `Credentials` "engine aware" to find the right default?